src/dom-to-image.js

```diff
@@ -137,8 +138,16 @@
             canvas.width = options.width || util.width(domNode);
             canvas.height = options.height || util.height(domNode);
 
+            var ctx = canvas.getContext('2d');
+
+            if (options.dpi != null) {
+              var scaleFactor = options.dpi / 96;
```

Comment:
My only concern is that 96 looks kinda like magic number which might change one day (I might be wrong). Is there any way out there to get current DPI through standard API?